app/models/prompt.rb

```diff
@@ -145,15 +147,17 @@ def allowed_tags
         elsif restriction.has_tags?(tag_type)
           disallowed_taglist = tag_set.send("#{tag_type}_taglist") - restriction.tags(tag_type)
           unless disallowed_taglist.empty?
-            errors.add(:base, ts("^These %{tag_type} tags in your %{prompt_type} are not allowed in this challenge: %{taglist}",
+            tag_label = tag_type_label_name(tag_type)
```

Comment:
I think Hound is right. Instead of doing it here, we can do it on line 153 (which would need to be changed to `tag_label: tag_label` if we kept the variable assignment here). Same re: the Hound comment on line 184.